src/util.ts

```diff
@@ -0,0 +1,42 @@
+import * as vscode from "vscode";
+
+export function getConfig() {
```

Comment:
what is the return type here?

Reply:
the type is inferred from workspace getConfiguration